src/components/ec-navigation/ec-navigation.vue

```diff
@@ -1,5 +1,8 @@
 <template>
-  <ec-mobile-header v-if="isResponsive">
+  <ec-mobile-header
+    v-if="isResponsive && !isMobileMenuOpen"
+    @open-mobile-menu="isMobileMenuOpen = true"
+  >
     <template #logo>
```

Comment:
I've been checking the preview and the logo size it's different depending on if the menu is open or not, is this okay?

![image](https://github.com/Ebury/chameleon/assets/77344353/9c1aa3cb-f7b7-418d-9cef-e51f402fb3a2)
![image](https://github.com/Ebury/chameleon/assets/77344353/fc574c33-8aca-4d0f-a51b-952067cd244a)

Reply:
Now it's not aligned with the close button
![image](https://github.com/Ebury/chameleon/assets/77344353/09eff766-93b0-488f-a6f2-0204f4b23bcd)

Reply:
Ok pushed some changes!